### DISCUSSION TITLE
[5.1] Missing size attributes for language flags

### DISCUSSION THREAD

**Discussion by M-u-e-c-k-e** [joomla/joomla-cms#47029] [2. Cold Features]
### System information (as much as possible)
Joomla 5.1.4
Templates (Site): Cassiopeia

### Multilingualism: Images: Missing size attributes  (Mehrsprachigkeit: Bilder: Fehlende Größenattribute)
The site has the languages German & English 

During the test (SEO) with[Screaming Frog SEO Spider](https://www.screamingfrog.co.uk/seo-spider/) however, the images of the language module were criticised.

Pictures: Missing size attributes

Solution The image size of the flags should also be entered in the code.

![Image](https://github.com/user-attachments/assets/1569598a-95e7-4ab6-82e8-3b0c1b99daa1)

![Image](https://github.com/user-attachments/assets/0716661e-c2c0-4e94-a08b-aa8debb683f2)

**Comment by ghost** [comment]
Can you change the german- to english-sentences in title and description?

**Comment by brianteeman** [comment]
> Including height and [width](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#width) enables the [aspect ratio](https://developer.mozilla.org/en-US/docs/Glossary/Aspect_ratio) of the image to be calculated by the browser prior to the image being loaded. This aspect ratio is used to reserve the space needed to display the image, reducing or even preventing a layout shift when the image is downloaded and painted to the screen. Reducing layout shift is a major component of good user experience and web performance.
Source [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#height)
